src/draive/openai/lmm_session.py

```
@@ -1,301 +1,353 @@
+                                ctx.record(
+                                    ObservabilityLevel.INFO,
+                                    metric="lmm.output_tokens",
+                                    value=usage.output_tokens
+                                    if usage.output_tokens is not None
+                                    else 0,
+                                    unit="tokens",
+                                    attributes={"lmm.model": config.model},
+                                )
+
+                            if event.response.status == "incomplete":
+                                continue  # TODO: handle incomplete if needed
```

Pylint (3.3.7)

[warning] TODO: handle incomplete if needed

(W0511)